model/dd2d/pario_nc.f

```diff
@@ -778,7 +778,12 @@ subroutine stoprc(rc,rc_ok)
       integer :: rc,rc_ok
       integer :: mpi_err
 #ifndef SERIAL_MODE
-      call mpi_bcast(rc,1,MPI_INTEGER,0,MPI_COMM_WORLD,mpi_err)
+      ! FIXME: Why does a call to mpi_bcast or mpi_barrier hang here in the
+      !        following call in model/IO_DRV.f?
+      !          call write_data(grid, fid, 'cputime', timing(1:ntimemax))
+      !
+      ! call mpi_barrier(MPI_COMM_WORLD,mpi_err)
+      ! call mpi_bcast(rc,1,MPI_INTEGER,0,MPI_COMM_WORLD,mpi_err)
```

Comment:
With this temporary hack workaround, the model does proceed successfully. However, it assumes that the NetCDF file was loaded successfully on the root MPI rank, which is clearly not always going to be the case.